profiles/Jonas-setup.yaml

```diff
@@ -1,15 +1,13 @@
 ---
-safe_room: 1645 # Crossing - Spooky Tree ooOoOoO
-# safe_room: 2855 # Shard - Dark Clearing
-# safe_room: 2717 # Shard - Ironwood Tree
-# safe_room: 2715 # Shard - Old Mine, Bar in Blackthorn Canyon (has rocks!)
+# safe_room: 1645 # Crossing - Spooky Tree ooOoOoO
+safe_room: 2715 # Shard - Old Mine, Bar in Blackthorn Canyon (has rocks!)
 # Hunting Settings
 
-hometown: Crossing
+hometown: Shard
 slack_username: jonas
+climbing_target: undergondola_pit
```

Comment:
There is a single `climbing_target`. Would you ever want to list multiple targets here (i.e. a list)?